src/main/java/dan200/computercraft/shared/computer/apis/CommandAPI.java

```diff
@@ -98,6 +99,7 @@ public void shutdown()
             }
             catch( Throwable t )
             {
+                ComputerCraft.log.error( "Error running command.", t );
```

Comment:
Ahhh, I wasn't aware of that. I'm not sure what the suitable compromise between "provide as much useful information to the developer as possible" and "don't spam the log" is. I'd personally err on more useful information with the ability to turn it off. 

From what I can tell from previous reports of log spam issue, it has been an incessant thing - this will only be a problem if someone can find a method which errors which is *generally* are rare thing.

---

A logging cooldown would be possible, though I'm not entirely sure if the implementation/benefit trade off is worth it.